src/api/forms/components/controls/AutoCompleteField.vue

```diff
@@ -155,11 +165,14 @@ export default {
         },
         inputClicked() {
             this.autocompleteInputElement.select();
-            this.showOptions(this.autocompleteInputElement.value);
+            this.showOptions();
         },
         arrowClicked() {
+            // if the user clicked the arrow, we want
+            // to show them all the options
+            this.showFilteredOptions = false;
             this.autocompleteInputElement.select();
-            this.showOptions('');
+            this.showOptions();
```

Comment:
wondering if we really need to call `this.showOptions();` (same on inputClicked)  I think reactive `showFilteredOptions` change takes care of `filteredOptions`? Can you confirm?